### ISSUE TITLE
Size - Storeys Changes

### ISSUE THREAD

**Issue by mdsimpson42** [colouring-cities/colouring-core#1192]
- [ ] Change stories so they are blank by default (instead of being set to zero). 
- [ ] Add a field "total stories" that automatically sums up the other storeys fields to give a total sum.

**Comment by mdsimpson42** [comment]
The values for the storeys appear to be set to '0' in the database. To reset them all to 'NULL', we would have to run a script on the database to delete all zeroes and replace them with 'NULL'. Might lose data....

**Comment by mdsimpson42** [comment]
This wouldn't affect our ability to sum up the number of floors (the sum would just be '0'). We could not display the sum value if the value was zero?

**Comment by mdsimpson42** [comment]
I have added code to calculate the total stories. Includes a check for null data, even though we don't have any null data yet.
("Null" is just programmer speak for blank/empty, it will simply appear as an empty box to the user)

**Comment by mdsimpson42** [comment]
Added this to the database label. Need to write a script to go through the database and set any values for storeys from '0' to Null. The only problem might be that if the building has no basement/attic, then the value for "floors within roof space"/"floors beneath ground Level" might have been set to 0 deliberately and we might lose data?

**Comment by polly64** [comment]
@matkoniecz any thoughts on this one? Also is there another word commonly used other than 'null' to indicate that no data has been entered yet?

**Comment by matkoniecz** [comment]
> Also is there another word commonly used other than 'null' to indicate that no data has been entered yet?

In common use? I would go with "no data"

In internal code/tech/database design? Null is relatively clear, though explicit "no data" also works

In internal of database? Null value is proper and standard solution

**Comment by polly64** [comment]
@matkoniecz yes very good let's go with 'no data' - are you able to fix this one?
